### PR TITLE
Min length and MaxLength ported from lc-form-validation

### PR DESCRIPTION
**Pull request by brauliodiez**

### CODE REVIEW

File: src/validators/max-length.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { LengthArgs, parseLengthParams, isLengthValid } from './length';
+import { FieldValidationFunctionSync } from '../model';
+
+export const VALIDATOR_TYPE = 'MAX_LENGTH';
+
+let defaultMessage = 'The value provided does not fulfil max length';
```
Comment (JaimeSalas) [inline src/validators/max-length.ts]: Typo 'fulfill'

File: src/validators/max-length.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export const setErrorMessage = message => (defaultMessage = message);
+
+const BAD_PARAMETER =
+  'FieldValidationError: Parameter "length" for maxLength in customArgs is mandatory and should be a valid number. Example: { length: 4 }.';
+
+const DEFAULT_PARAMS: LengthArgs = null;
+
+function isStringLengthValid(value: string, length: number): boolean {
+  return value.length <= length;
+}
```
Comment (JaimeSalas) [inline src/validators/max-length.ts]: Why not fat arrow anonymous function for isStringLengthValid?

File: src/validators/max-length.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+export const maxLength: FieldValidationFunctionSync = fieldValidatorArgs => {
+  if (!fieldValidatorArgs.customArgs) {
```
Comment (JaimeSalas) [inline src/validators/max-length.ts]: Missed return type?

File: src/validators/min-length.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import { LengthArgs, parseLengthParams, isLengthValid } from './length';
+import { FieldValidationFunctionSync } from '../model';
```
Comment (JaimeSalas) [inline src/validators/min-length.ts]: Could be a good idea to have common base code for min and max validations?

File: src/validators/length.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  length: number;
+}
+
+export function parseLengthParams(
+  customParams: LengthArgs,
+  errorMessage: string
+) {
+  const length =
+    customParams.length === null ? NaN : Number(customParams.length);
+  if (isNaN(length)) {
```
Comment (JaimeSalas) [inline src/validators/length.ts]: Is this casting Number(customParams.length); necessary?

Comment (brauliodiez) [reply src/validators/length.ts]: I think is defensive programming if somebody by mistake enter a non numeric length number it just throws an error (was on the original validator I have ported)

File: src/validators/max-length.ts (hunk 1 above)
Comment (brauliodiez) [reply src/validators/max-length.ts]: fixed, thanks

File: src/validators/max-length.ts (hunk 2 above)
Comment (brauliodiez) [reply src/validators/max-length.ts]: Ported from the other project, I think is a good idea to port it to fat arrow, going to udpated it., Done

File: src/validators/max-length.ts (hunk 3 above)
Comment (brauliodiez) [reply src/validators/max-length.ts]: Sorry don't get it, could you please ellaborate?

File: src/validators/min-length.ts (hunk 4 above)
Comment (brauliodiez) [reply src/validators/min-length.ts]: There is already a file length.js that already contains some common goodies, I was tempted to extract more stuff but decided to stop, the reason why: if somebody hops into this code and takes it as reference to create his own validator it should be something readable (almost all the validators follow the same pattern), what do you think?

**Pull request merged**